include/vara/Configuration/Configuration.h

```diff
@@ -0,0 +1,50 @@
+#ifndef VARA_CONFIGURATION_CONFIGURATION_H
+#define VARA_CONFIGURATION_CONFIGURATION_H
+
+#include "vara/Configuration/ConfigurationOption.h"
+
+namespace vara::feature {
+
+/// \brief This class represents a configuration.
+/// The configuration is represented by using a json format of the form
+/// \c `{name: value, name2: value2}`.
+class Configuration {
+public:
+  explicit Configuration() = default;
```

Comment:
```suggestion
  Configuration() = default;
```
`explicit` is mostly important for ctors that take 1 parameter, but I don't think we need it in this case here.